notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1623 @@
+{
```

Comment:
<p>very unimportant nit - is there a way in notebooks that you can add a hyperlink to "Switch your kernel" so that when the user clicks on it you snap to that header? Like you can do in Markdown? If so, that would be a nice thing to add here. But it's very much a "would be nice" thing. Don't spend more than like 10 minutes trying to figure it out.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I tried in my first project but there is no way to add a hyperlink for it or switch the kernel itself :(</p>